plugins/aws_sts/README.md

```diff
@@ -0,0 +1,20 @@
+# AWS STS Plugin
+
+Inject temporary AWS Role credentials into a deploy environment using [AWS STS](https://docs.aws.amazon.com/STS/latest/APIReference/Welcome.html) and [Assume role](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html).
+
+## Overview
+
+Allows Samson to inject temporary security credentials for an AWS Role. The plugin uses
+the assume role feature of STS and exposes the generated credentials as environment variables:
+  - STS_AWS_ACCESS_KEY_ID
+  - STS_AWS_SECRET_ACCESS_KEY
+  - STS_AWS_SESSION_TOKEN
+
+The plugin authenticates with AWS using the following environment variables:
+  - SAMSON_STS_AWS_ACCESS_KEY_ID
+  - SAMSON_STS_AWS_SECRET_ACCESS_KEY
+  - SAMSON_STS_AWS_REGION
+
+These should be configured in the secret manager against the deploy group or stage using the plugin.
```

Comment:
this sounds like they would be configured per project ... maybe just remove this part and hope people know how to add secrets to samson :)